runtime/lua/vim/lsp/diagnostic.lua

```diff
@@ -79,7 +79,7 @@ end
 --- @return table?
 local function tags_lsp_to_vim(diagnostic, client_id)
   local tags ---@type table?
-  for _, tag in ipairs(diagnostic.tags or {}) do
+  for _, tag in ipairs(diagnostic._tags or {}) do
```

Comment:
This is an lsp diagnostic, this should be kept as `tags`.

Reply:
Done 👍🏾